jennchenn/uwexplore
# Prevent users from saving duplicate courses to calendar

## Implementation description

- Add check on section id before user adds a new course to their calendar to ensure it's unique



## Steps to test

1. Add a course to schedule
```
curl -H "Content-Type: application/json" -X POST -d '{"course_id":"64126b46269911f00d6f1db8", "section_id":"64126ec4269911f00d6f33d2","color":"ffffff"}'  "localhost:5000/courses/schedules/64127ede93deee8bdc7a9121"
```
2. Try to add the same course
```
curl -H "Content-Type: application/json" -X POST -d '{"course_id":"64126b46269911f00d6f1db8", "section_id":"64126ec4269911f00d6f33d2","color":"ffffff"}'  "localhost:5000/courses/schedules/64127ede93deee8bdc7a9121"
```
3. Verify it errors out and course isn't added



## What should reviewers focus on?

- there is no return if there's error
